Marlin/src/inc/Conditionals_post.h

```diff
@@ -2504,9 +2504,15 @@
 // PID heating
 #if ANY(PIDTEMP, PIDTEMPBED, PIDTEMPCHAMBER)
   #define HAS_PID_HEATING 1
-  #if ENABLED(DWIN_LCD_PROUI) && EITHER(PIDTEMP, PIDTEMPBED)
+#endif
+
+#if ENABLED(DWIN_LCD_PROUI)
+  #if EITHER(PIDTEMP, PIDTEMPBED)
     #define DWIN_PID_TUNE 1
   #endif
+  #if EITHER(DWIN_PID_TUNE, MPCTEMP)
+    #define HAS_PIDPLOT 1
```

Comment:
Note that HAS_PIDPLOT can be disabled to save program memory, useful for the Ender3 S1 with F4 SoC, due to less program memory available on that board.

Reply:
I would expect that to be the case. But I needed to define it for testing.

Reply:
Since the plot is also displayed for MPC, perhaps a better name than HAS_**PID**PLOT could be recommended.

Reply:
When it becomes a configuration option it should lose the `HAS_` because that prefix is an indicator of a conditional. So I'll try on a few new names and see what fits best.